Documentation/Contributors/BuildGuide/README.md

```diff
@@ -76,38 +94,43 @@ npm run [target-name]
-   * `deploy-s3` - Deploys the built cesium files, the npm package, and the zip file to Amazon S3. This requires having credentials set up for the S3 bucket to which you are deploying.
-   * `deploy-status` - Set the deployment statuses in GitHub, for use with Travis.
-   * `deploy-set-version` - Sets the version of `package.json`, for use with Travis.
+
+   * **Build scripts** -- build and package the source code (and documentation)
+      * `build` - A fast, developer-oriented build that prepares the source tree for use as standard [Asynchronous Module Definition (AMD)](https://github.com/amdjs/amdjs-api/wiki/AMD) modules, suitable for running tests and most examples (some Sandcastle examples require running `combine`).  This runs automatically when saving files in Eclipse.
+      * `build-watch` - A never-ending task that watches your file system for changes to Cesium and runs `build` on the source code as needed. 
+      * `combine` - Runs `build`, plus the [the RequireJS optimizer](http://requirejs.org/docs/optimization.html) to combine Cesium and [the Almond AMD loader](http://requirejs.org/docs/faq-optimization.html#wrap) to produce all-in-one files in the `Build/Cesium` directory that expose the entire Cesium API attached to a single global Cesium object.  This version is useful if you don't want to use the modules directly with a standard AMD loader.
+      * `minify` - Runs `combine`, plus [minifies](http://en.wikipedia.org/wiki/Minification_\(programming\)) Cesium.js using [UglifyJS2](https://github.com/mishoo/UglifyJS2) for a smaller deployable file.
+      * `combineRelease` - Runs `combine`, plus uses the optimizer to remove debugging code that validates function input and throws DeveloperErrors.  The removed sections are marked with `//>>includeStart('debug', pragmas.debug);` blocks in the code.
+      * `minifyRelease` - Runs `minify`, and removes debugging code.
+      * `requirejs` - Used internally by the build system and can not be called directly.
+      * `buildApps` - Builds the example applications (such as Cesium Viewer) to produce self-contained, minified, deployable versions in the `Build` directory.
+      * `generateDocumentation` - Generates HTML documentation in `Build/Documentation` using [JSDoc 3](https://github.com/jsdoc3/jsdoc). More [details here](https://github.com/rahwang/cesium/tree/master/Documentation/Contributors/DocumentationGuide).
+      * `release` - A full release build that creates a shippable product, including building apps and generating documentation.
+   * **Utility scripts** -- crucial development tools
+      * `instrumentForCoverage` - Runs [JSCoverage](http://siliconforks.com/jscoverage/) on the source tree to allow running tests with coverage information.  Use the link in index.html.  Currently Windows only.
+      * `jsHint` - Runs [JSHint](http://www.jshint.com/), a helpful code analysis tool, on the entire source tree.
```

Comment:
Avoid empty terms like "helpful."

"helpful code analysis tool" could perhaps be "static code analysis tool"